Update cellDuration description in Scheduler article (#1635)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxScheduler/1 Configuration/cellDuration.md

---
---
##### shortDescription

> Specifies cell duration in minutes. The interval between [startDayHour]({basewidgetpath}/Configuration/#startDayHour) and [endDayHour]({basewidgetpath}/Configuration/#endDayHour) should be divisible by this value.

Reviewer: Hello. Please do not use {basewidgetpath} for links in doc members that belong to a certain UI component. In this case, replace them with links that lead to the Scheduler startDayHour and endDayHour properties.

Author: done

> Specifies cell duration in minutes. The interval between [startDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#endDayHour) should be divisible by this value.

Reviewer: I also suggest you to get rid of passive voice as follows:
```suggestion
Specifies cell duration in minutes. This property's value should divide the interval between [startDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#endDayHour) into even parts.
```

> Specifies cell duration in minutes. This property's value should divide the interval between [startDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#endDayHour) into even parts.

Reviewer 2: ```suggestion
Specifies cell duration in minutes. This property's value should evenly divide the interval between [startDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#startDayHour) and [endDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#endDayHour).
```

Reviewer 2: What do you mean by "cell duration"? Can you explain?

Author: Cell duration denotes the amount in minutes allocated per cell. This value affects the visual representation of cells within the Scheduler.

Reviewer 2: Ok
